### ISSUE TITLE
Decide on gas parameters

### ISSUE THREAD

**Issue by eyusufatik** [chainwayxyz/citrea#453]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by rakanalh** [comment]
Optimisim:
- Minimum Base Fee: 1 GWEI
- Elasticity Multiplier: 10
- Base Fee Max Change Denominator: 8
- Ref: [https://github.com/ethereum-optimism/optimism/blob/54eb4161dc0d2dad7e6840b788ef9704a0f5c721/packages/core-utils/src/optimism/constants.ts#L34](Link 1), [https://docs.optimism.io/chain/differences#eip-1559-parameters](Link 2)

Arbitrum:
- Minimum Base Fee: 1 GWEI
- Elasticity Multiplier: 2
- Base Fee Max Change Denominator: 8
- Ref: [https://github.com/OffchainLabs/go-ethereum/blob/08bf9b119ff9c47ad51d507969997ff9cd8c5328/params/protocol_params.go#L127](Link)

ZkSync (Foundary):
- Minimum Base Fee: 1 GWEI
- Elasticity Multiplier: 2 ((1/8 in the second link seems to be what's actually used, see Link 2)
- Base Fee Max Change Denominator: 8 
- Ref: [https://github.com/matter-labs/foundry-zksync/blob/5ae06f94bcc3f6588955646fba22694dd0ebc831/crates/anvil/src/eth/fees.rs#L29](Link), [https://github.com/matter-labs/foundry-zksync/blob/5ae06f94bcc3f6588955646fba22694dd0ebc831/crates/anvil/src/eth/fees.rs#L34](Link 2)

Polygon ZkEVM:
- Minimum Base Fee: 1 GWEI
- Elasticity Multiplier: 2
- Base Fee Max Change Denominator: 8
- Ref: [https://github.com/0xPolygonHermez/cdk-erigon/blob/1d56fb0a5a64160fd8c05e11ffc8b668bd70b9e8/params/protocol_params.go#L121](Link)

Linea:
- Minimum Base Fee: 1 GWEI
- Elasticity Multiplier: 2
- Base Fee Max Change Denominator: 8
- Ref: [https://github.com/Consensys/linea-besu/blob/a5a3eb8a36c119011e6ed1988bdb76af6ec7e304/ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/feemarket/LondonFeeMarket.java#L33](Link)

In short:
- They all use the same minimum base fee
- They all use the same max change denominator
- Elasticity multiplier differs between those L2s, no information as to why they chose such a value.

**Comment by rakanalh** [comment]
Given this calculation:
https://github.com/paradigmxyz/reth/blob/c04dbe6e9bd05be5da3a5d541adbf76166c14a08/crates/primitives/src/basefee.rs#L26-L62

Created a simulation on google sheets and noticed that the smaller the values we set for `elasticity_multiplier` and `max_base_fee_change_denominator`, the higher the base fee delta will be which means that the transactions will get more expensive on congested blocks compared to when those parameters use higher values.

Decision with @eyusufatik was not to update the default ethereum values we use for now.
